modules/local/sanger_tol_btk.nf

```diff
@@ -0,0 +1,101 @@
+process SANGER_TOL_BTK {
+    tag "$meta.id"
+    label 'process_low'
+
+    input:
```

Comment:
There should be an input for a comma separated list of BUSCO lineages that will be used in the run, used as the  `--busco_lineages` argument in the `draft_assemblies` branch of `sanger-tol/blobtoolkit`

Reply:
This should be added now, just needs some testing